manifests/php.yml

```diff
@@ -550,10 +550,10 @@ tests/:
     Test_Config_HttpServerErrorStatuses_FeatureFlagCustom: missing_feature
     Test_Config_IntegrationEnabled_False: v1.4.0
     Test_Config_IntegrationEnabled_True: v1.4.0
-    Test_Config_LogInjection_128Bit_TraceId_Disabled: incomplete_test_app (endpoint not implemented)
-    Test_Config_LogInjection_128Bit_TraceId_Enabled: missing_feature (not enabled by default)
-    Test_Config_LogInjection_Default: incomplete_test_app (endpoint not implemented)
-    Test_Config_LogInjection_Enabled: incomplete_test_app (endpoint not implemented)
+    Test_Config_LogInjection_128Bit_TraceId_Disabled: incomplete_test_app (/log-library endpoints fails with a 500, this needs a deeper investigation)
```

Comment:
Can you tell me more about this?

Reply:
We have two failing tests
- Ahhhh I think my monolog endpoint is using an import (`use Monolog\Level`) is that was introduced in `monolog v3.0`, and older php versions (php<=8.1) require monolog<2.0. I'll update this endpoint.